server/plugin/plugin.go

```diff
@@ -119,3 +127,17 @@ func isValidTaskLink(msg string) ([]string, bool) {
 	}
 	return data, true
 }
+
+// Function to validate the pull request link.
+func isValidPullRequestLink(msg string) ([]string, string, bool) {
+	pullRequestRegex := regexp.MustCompile(constants.PullRequestRegex)
+	link := pullRequestRegex.FindString(msg)
+	data := strings.Split(link, "/")
+	if len(data) != 9 {
+		return nil, "", false
+	}
+	if (data[0] != constants.HTTPS && data[0] != constants.HTTP) || data[2] != constants.AzureDevopsBaseURL || data[5] != constants.Git || data[7] != constants.PullRequest {
```

Comment:
we are checking the URL by comparing it with regex. do we need to explicitly check it like this?